Runtime/ApiClient.cs

```diff
@@ -118,25 +118,25 @@ public async UniTask<T> Delete<T>(string path, object body = null, Dictionary<st
 		/// <param name="token">cancellation token</param>
 		/// <typeparam name="T">A <see cref="System.Serializable" /> class to populate with data. </typeparam>
 		/// <returns>T</returns>
-		public async UniTask<T> Patch<T>(string path, object body = null, Dictionary<string, string> headers = null, CancellationToken token = default)
+		public async UniTask<HttpResponseObject<T>> Patch<T>(string path, object body = null, Dictionary<string, string> headers = null, CancellationToken token = default)
 		{
 			return await Request<T>(HttpMethod.Patch, path, headers, body, token: token);
 		}
 
```

Comment:
Any reason why these are commented out?

Reply:
We aren't using them, they are left over from the code we used as a basis for our HTTP stuff.